pokemongo_bot/cell_workers/move_to_fort.py

```diff
@@ -109,7 +109,7 @@ def work(self):
                     formatted='Arrived at fort.'
                 )
 
-        return WorkerResult.SUCCESS
```

Comment:
There is no point but people will still do it
>There is no point to put your catch Pokemon task after the moving task. 

The point is less opened issues to spend our time with...
but if you want to "educate" users on how to configure the bot and what is the "supported" task issues and after that  look at all issues rising from this.. we'll i guest you do have a lot of time to spare..
>There is no point offering this kind of flexibility